protected/humhub/modules/content/widgets/stream/WallStreamEntryWidget.php

```diff
@@ -367,4 +367,13 @@ public function getAttributes()
             'class' => $this->renderOptions->isPinned($this->model) ? 'wall-entry pinned-entry' : 'wall-entry'
         ];
     }
+
+    /**
+     * Register assets that are required for menu item of content creating
+     *
+     * @since 1.13
+     */
+    public function registerCreateContentMenuAssets()
+    {
```

Comment:
In case we really need that, i.e. the assets can't be loaded in the Create modal. Would it be good to add a link to the Tasks/Calendar module here for the documentation, so you can see how it can be used.

Reply:
It was deleted here https://github.com/humhub/humhub/pull/5827/commits/7632d9cc15fbbfeb10a97bd91ac04eee93099996